AirSkyBoat/AirSkyBoat
# Fix nuke wall incorrectly applying to certain spells

**_I affirm:_**


## Please enter a player-facing description
- The nuke wall will no longer be incorrectly applied to certain non-damaging spells like enfeebling spells. (Tracent)
- The following spells have had their base damage slightly adjusted to match era values (Aero IV +30, Tornado -105, Blizzard IV -35, Freeze -77, Flood +53). (Tracent)

## What does this pull request do? (Please be technical)
This fixes an issue whereby the nuke wall was being incremented and applied to non damaging spells. The incrementing was due to misuse of ternary (which still evaluates both potential outputs no matter the conditional, thus if these output functions have side effects then they are also applied). While the application was due to missing check for isDamageSpell. Also renamed the nuke wall functions to be more descriptive and make the code less confusing and removed an unused parameter isEnfeeb.

Also adjusted several spell base damage to fix what looks like copy-paste errors, for reference the values come from [here](https://web.archive.org/web/20080615010322/https://wiki.ffo.jp/html/3199.html).

## Steps to test these changes
On an NM use chainspell and cast a large number of earth spells, for example, and then try casting slow on the NM. The slow spell should land like normal.

## Special Deployment Considerations
